client/src/app/views/form.coffee

```diff
@@ -0,0 +1,28 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+template = require('../templates/form.jade')
+errorMessage = require('../templates/error.jade')
+SiteModel = require('../models/site')
+
+module.exports = Marionette.ItemView.extend(
+  model: new SiteModel()
+  template: template
+  events:
+    'submit form' : 'enterSite'
+  ui:
+    url: '#url'
+    email: '#email'
+  enterSite: (e) ->
+    e.preventDefault()
+    unless @validateForm() then return
+    url = @ui.url.val().trim()
+    @ui.url.val('')
+    @ui.email.val('')
+  validateForm: ->
```

Comment:
lets add a blank line between hash groups and function definitions just for readability